### ISSUE TITLE
Sub Parameter Values Left Stale & New Parameter Created After Multi Byte Parameter Update

### ISSUE THREAD

**Issue by The00Dustin** [zwave-js/zwave-js-ui#3651]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Home Assistant Add-on

### Z-Wave JS UI version

9.9.1

### ZwaveJS version

12.4.4

### Describe the bug

When, using a HomeAssistant service, I send a multi byte value to a multi byte parameter that is listed in Z-Wave JS UI as separate sub parameters, Z-Wave JS UI adds a new parameter for that value update and leaves the sub parameters stale.  If a sub parameter is updated afterwards, the multi byte parameter will be stale.  Refreshing values will correct the sub parameters, but not the multi byte parameter.

### To Reproduce

Review the end of the configuration parameters before and after taking the step mentioned in the description.  In the reproduction steps I took for creating logs for this issue, I used the following service call:
```
service: zwave_js.bulk_set_partial_config_parameters
data:
  parameter: "99"
  value: 25191679
target:
  entity_id: light.media_light
```

### Expected behavior

The sub parameters should have their values updated from the multi-byte event, and *if* the multi-byte parameter should also exist, it should also update when notified of changes to the sub parameters and during refreshing values.

### Additional context

Here is a screen snippet before sending the multi-byte parameter value:
![Screenshot 2024-03-28 at 13 12 31](https://github.com/zwave-js/zwave-js-ui/assets/91274634/fd791528-675b-478b-845e-ae9f7651e46a)

Here is a screen snippet after sending the multi-byte parameter value:
![Screenshot 2024-03-28 at 13 20 21](https://github.com/zwave-js/zwave-js-ui/assets/91274634/0c3daea9-e915-43f6-8f39-3aea5531aa1e)

Here are the logs:
[zwave-js-ui-store.zip](https://github.com/zwave-js/zwave-js-ui/files/14793505/zwave-js-ui-store.zip)

Here are UTC timestamps to go with the logs (Node 12):
17:11 - Reinterview
17:17 - Send multi byte value - see that new property is created & old sub parameters aren't updated
17:25 - Refresh values - see that sub parameters are now all updated
17:26 - Send single byte value (with Z-Wave JS UI) - see that new property isn't updated

*I did more testing after 17:26 repeating the last two steps & sending the single byte value with the service, but didn't see any differences in behavior, the new multi byte parameter stayed the same as shown in the second screen snippet.

**Comment by robertsLando** [comment]
I don't think this is something related to Z-UI side, maybe an event missing from Z-Wave JS? It's not 100% clear what you are doing here as I never did such operation, @AlCalzone any idea?

**Comment by AlCalzone** [comment]
Yeah this was supposed to be opened in the node-zwave-js repo

**Comment by The00Dustin** [comment]
Sorry about that, makes sense now that I think about it. I see the sub-parameters in UI, but I guess they are technically exposed to HA, which puts them in node.  Should I close and re-open this, or can it be moved?

**Comment by AlCalzone** [comment]
I'll move it, it's fine. If something is missing that the other issue template asks for, I'll get back to you.
